Makefile

```diff
@@ -1,33 +1,39 @@
 CXX ?= g++
-CCFLAGS=-g -Wall -Werror -Wformat=2 -Wextra -Wwrite-strings \
+CC ?= gcc
+CXXFLAGS=-g -Wall -Werror -Wformat=2 -Wextra -Wwrite-strings \
 -Wno-unused-parameter -Wmissing-format-attribute -Wno-non-template-friend \
 -Woverloaded-virtual -Wcast-qual -Wcast-align -Wconversion -fomit-frame-pointer \
 -std=c++11 -fPIC -O3
+CFLAGS=-g -Wall -Werror -Wformat=2 -Wextra -Wwrite-strings \
+-Wno-unused-parameter -Wmissing-format-attribute \
+-Wcast-align -Wconversion -fomit-frame-pointer \
+-std=gnu99 -fPIC -O3 -lstdc++
 
 # Output directories
 OBJECT_DIR = obj
 SRC_DIR = src
+WRAPPER_DIR = cwrapper
 INCLUDE_DIR = include/Arachne
 LIB_DIR = lib
 BIN_DIR = bin
 
 # Depenencies
 PERFUTILS=../PerfUtils
 COREARBITER=../CoreArbiter
-INCLUDE=-I$(PERFUTILS)/include -I$(COREARBITER)/include
+INCLUDE=-I$(PERFUTILS)/include -I$(COREARBITER)/include -I$(INCLUDE_DIR)
```

Comment:
s/INCLUDE_DIR/SRC_DIR/

This removes the circular dependency which causes constant rebuilds of `obj/CArachneWrapper.o`.